crates/wysiwyg/src/dom/dom_block_nodes.rs

```diff
@@ -86,7 +86,8 @@ where
                 last_ancestor_to_split.depth(),
             );
             let mut min_depth = 0;
-            for i in min_depth..max_depth {
+            let start_depth = min_depth;
+            for i in start_depth..max_depth {
```

Comment:
Here clippy didn't like the fact that the name of the variable changed on 91 is the same as that which is the start of this range. Just separated them into separate variables to get around that.